recipe/meta.yaml

```diff
@@ -54,15 +53,17 @@ outputs:
     build:
       noarch: python
     requirements:
-      - python >=3.9
-      - bokeh >=3.4.0,<3.5.0
+      - python >=3.10
```

Comment:
Just a thought: do you think it's required to list these dependencies again when `{{ pin_subpackage('geoviews-core', exact=True) }}` is included?